lib/plugins/aws/invokeLocal/invoke.rb

```diff
@@ -0,0 +1,67 @@
+require 'json'
+
+class FakeLambdaContext
+  attr_reader :function_name, :function_version
+
+  def initialize(name: 'Fake', version: 'LATEST', timeout: 6, **options)
+    @function_name = name
+    @function_version = version
+    @created_time = Time.now()
+    @timeout = timeout
+    options.each {|k,v|
+      send(k, v)
+    }
+  end
+
+  def get_remaining_time_in_millis
+    [@timeout*1000 - ((Time.now() - @created_time)*1000).round, 0].max
+  end
+
+  def invoked_function_arn
+    "arn:aws:lambda:serverless:#{function_name}"
+  end
+
+  def memory_limit_in_mb
+    return '1024'
```

Comment:
@dholdren minor nitpick - remove the unnecessary `return` from following 4 methods.